<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoryForm</class>
 <widget class="QWidget" name="CategoryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>311</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDelete">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCode">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>53</y>
      <width>125</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>84</y>
      <width>125</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>52</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>85</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelCodeValid">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>85</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Code may be empty or contains text or greater than 10000</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameValid">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>125</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name may be empty or exceeds 20 characters</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditCode</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonDelete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
